src/core/data.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
     },
 
     readFile(filePath) {
-        const format = utils.lang(filePath, true).mode;
+        const format = utils.lang(filePath, true).ace_mode;
```

Comment:
I noticed there's also a reference to `utils.lang(filePath, true).mode` on [line 77](https://github.com/frctl/fractal/blob/6831f8000a5ffb9f8ba7d0977f9320dc0d198f49/src/core/data.js#L77).

Reply:
You're right, I don't know where it's used but I missed that. Going to fix this right now.

Reply:
Fixed. @ten1seven Btw.: Are you experiencing [this issue](https://github.com/frctl/fractal/issues/160#issuecomment-330337482) too?